cumulus/pallets/executive/src/lib.rs

```diff
@@ -170,7 +170,6 @@ impl<
 	OriginOf<Block::Extrinsic, Context>: From<Option<System::AccountId>>,
 	UnsignedValidator: ValidateUnsigned<Call = CallOf<Block::Extrinsic, Context>>,
 {
-	#[allow(unconditional_recursion)]
```

Comment:
The lint warning still persists though.

Reply:
Where? CI has passed without warnings with this suppression removed.

Reply:
You are right. Could be cache or something falsely reported by my editor, now it's fine.